### ISSUE TITLE
Captcha solver no longer works

### ISSUE THREAD

**Issue by bungholemaster** [K1rakishou/Kuroba-Experimental#982]
![Screenshot_20240111_125951](https://github.com/K1rakishou/Kuroba-Experimental/assets/16147700/91fc9edc-2668-4af3-84c6-77b4e3801af9)

They changed the captchas again, and now they are no longer solved accurately

**Comment by XPEH69** [comment]
Yeah, just tried posting and it got every captcha wrong. They didn't even change it that much, so I'm assuming Dimon can fix it faster than last time. Just keep captcha donation enabled

**Comment by dogsonbobnut** [comment]
I'm experiencing the same issue from a few days ago where I can access captcha at all after seeing the auto solver do this all day

**Comment by K1rakishou** [comment]
Did you try to update the solver? Because I updated the model to the latest one yesterday.
https://github.com/K1rakishou/4chanCaptchaSolver/releases/tag/v3.1

**Comment by bungholemaster** [comment]
> Did you try to update the solver? Because I updated the model to the latest one yesterday. https://github.com/K1rakishou/4chanCaptchaSolver/releases/tag/v3.1

I have updated to 3.1 now, but it's still not entirely accurate

**Comment by dogsonbobnut** [comment]
I updated the solver, but forgot that kuroba only checks for it once at install, so I reinstalled kuroba after updating the solver. **No issues after this.**

**Comment by K1rakishou** [comment]
No you didn't need to reinstall it because it checks it only once per launch. You only had to restart it.
